hs/t/y/token_track_container.txt

```diff
@@ -3,3 +3,5 @@ Verifying for generic connector
   Verifying when ALL participants are honest
   Verifying when NO participants are honest
 Checked 116 theorems; No failures!
+WARNING: Compiler instructed to emit for Algorand, but the conservative analysis found these potential problems:
+ * Step 2 may use up to 10 transaction references, but the limit is 8. Step 2 starts at ./token_track_container.rsh:24:5:dot.
```

Comment:
Do you know why this appeared now?

Reply:
The dynamic token lookup used a safe addition which prevented optimization from happening; I changed it to a verified addition and fixed it. No assertions will be needed for the addition since the verified op is generated after Core

Reply:
This answer seems to say something that is the opposite of the compiler output. You're saying that you enabled an optimization, but this error is essentially saying that the program is too big.

Reply:
Ah, and now I see you're referring to a new commit